db/migrations/003_teams.sql

```diff
@@ -45,11 +45,15 @@ CREATE TABLE  IF NOT EXISTS team_components (
   team_id UUID NOT NULL,
   component_id UUID NOT NULL,
   role TEXT NULL,
+  selector_id TEXT,
   PRIMARY KEY (team_id, component_id),
   FOREIGN KEY (team_id) REFERENCES teams(id)
-  -- FOREIGN KEY (component_id) REFERENCES components(id)
+  FOREIGN KEY (component_id) REFERENCES components(id)
+  UNIQUE (team_id, component_id, selector_id)
 );
 
+INSERT into incident_commander_db_version(version_id, tstamp, is_applied) VALUES(6, NOW(), true);
```

Comment:
We can hold off the metadata increment handling till we move to v1 or ship to a customer, what to you think ? Saying it because the schema's are in constantly changing state till the MVD is ready

Reply:
I think thats fine - I want to avoid having to recreate the db - Just before v1 we can refactor down